### PR TITLE
Change MFA prompt, include wps in docs examples

### PR DESCRIPTION
**Pull request by ekinanp**

### CODE REVIEW

File: plugin/aws/profile.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ func newProfile(ctx context.Context, name string) (*profile, error) {
 
 	// profile-specific stdin prompt
 	tokenProvider := func() (string, error) {
-		return plugin.Prompt(fmt.Sprintf("Assume ROLE MFA token code for %v", name))
+		return plugin.Prompt(fmt.Sprintf("Enter ROLE MFA token code for %v", name))
```
Comment (MikaelSmith) [inline plugin/aws/profile.go]: This is clearer that it's a request. Might be clearer what you're enteringif if we said `Enter MFA token code for role %v`.

**Review by MikaelSmith — APPROVED** [review]

File: plugin/aws/profile.go (hunk 1 above)
Comment (ekinanp) [reply plugin/aws/profile.go]: Makes sense. Updated.

File: plugin/aws/profile.go [hunk 2]
```diff
@@ -29,7 +29,7 @@ func newProfile(ctx context.Context, name string) (*profile, error) {
 
 	// profile-specific stdin prompt
 	tokenProvider := func() (string, error) {
-		return plugin.Prompt(fmt.Sprintf("Assume ROLE MFA token code for %v", name))
+		return plugin.Prompt(fmt.Sprintf("Enter MFA token code for role %v", name))
```
Comment (MikaelSmith) [inline plugin/aws/profile.go]: Hmm, actually the name is of the profile. Maybe we need to mention role.

Comment (ekinanp) [reply plugin/aws/profile.go]: Updated again

**Review by MikaelSmith — APPROVED** [review]

File: plugin/aws/profile.go (hunk 2 above)
Comment (MikaelSmith) [reply plugin/aws/profile.go]: Ugh, I meant to say "don't need to", but what's here seems fine.

Comment (ekinanp) [reply plugin/aws/profile.go]: No worries. Updated again.

**Review by MikaelSmith — APPROVED** [review]

**Pull request merged**